napalm/junos/junos.py

```diff
@@ -669,12 +669,18 @@ def get_lldp_neighbors_detail(self, interface=''):
         # and rpc for M, MX, and T Series is get-lldp-interface-neighbors
         # ref1: https://apps.juniper.net/xmlapi/operTags.jsp  (Junos 13.1 and later)
         # ref2: https://www.juniper.net/documentation/en_US/junos12.3/information-products/topic-collections/junos-xml-ref-oper/index.html  (Junos 12.3) # noqa
+        # Exceptions:
+        # EX9208    personality = SWITCH    RPC: <get-lldp-interface-neighbors><interface-device>
         lldp_table.GET_RPC = 'get-lldp-interface-neighbors'
-        if self.device.facts.get('personality') not in ('MX', 'M', 'T'):
+        if 'EX9208' in self.device.facts.get('model'):
```

Comment:
Does this affect only `EX9208`, or the entire EX series?

Reply:
@mirceaulinic I cannot tell... unfortunately I only have one EX9208...

Reply:
Fair enough. If anyone else will encounter the same on a different EX device, they will probably report that.
Please comment and explain this (and note that if they use a different EX series having the same behaviour they should report / fix).

Reply:
I collected some more info:

- EX2200 running Junos 15 needs `get-lldp-interface-neighbors-information` (so there's no need to add it here)
- EX4500 again needs `get-lldp-interface-neighbors-information`
- QFX10008 however still has the old `get-lldp-interface-neighbors` so we should probably add it to this if statement.

I see this list of exceptions growing, Juniper really messed up with this. I will revisit this later and refactor a little of this code to avoid dozens of if-else. But for the moment it's okay to add QFX10008 here.